structlog_sentry/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import sys
+
+from sentry_sdk import capture_exception, capture_message
+
+
+class SentryProcessor:
+    """Sentry processor for structlog. Uses Sentry SDK to capture events in Sentry."""
+
+    def __init__(self, sentry_log_level=logging.ERROR):
+        self.sentry_log_level = sentry_log_level
+
+    def _log(self, event_dict, level="error"):
+        exc_info = event_dict.pop("exc_info", sys.exc_info())
+        has_exc_info = exc_info and exc_info != (None, None, None)
+
+        if has_exc_info and logging.getLevelName(level.upper()) >= logging.ERROR:
+            return capture_exception(error=exc_info)
+        return capture_message(event_dict.get("event", ""), level=level)
+
+    def __call__(self, logger, method, event_dict):
+        event_dict["sentry"] = "skipped"
+
+        sentry_skip = event_dict.pop("sentry_skip", False)
+        sentry_dsn = os.environ.get("SENTRY_DSN")
+
+        level = event_dict["level"]
+        do_log = logging.getLevelName(level.upper()) >= self.sentry_log_level
```

Comment:
`getLevelName` is returning level value from name by mistake. It was removed and added back in the past. It's still undocumented behaviour, which we should not rely on, as it can be removed anytime. https://docs.python.org/3/library/logging.html#logging.getLevelName

I would suggest to change it to `getattr(logging, level.upper())` which should work reliably forever.